### DISCUSSION TITLE
MyST - Relative HTML link

### DISCUSSION THREAD

**Discussion by andormarkus** [executablebooks/meta#497] [Q&A]
Hi All,

I would like to create relative HTML link in MyST:

 reST syntax is working:
````markdown
```{eval-rst}
`beaver <./beaver>`_ (CircleCI orbs)
```
````

Markdown syntax does not work
```markdown
[beaver](./beaver)** (CircleCI orbs)
```

but I got the following error:
```shell
/docs/docs/source/index.md:15: WARNING: 'myst' reference target not found: ./beaver
```

How can I solve this problem?

Thanks,
Andor

**Comment by choldgraf** [comment]
Could you try attaching the extension? "Beaver.md"?

**Reply by andormarkus** [reply]
Every repo uses separated Sphinx-doc, and I got the following repo setup:

- repo "owl" - core documentation repo and serves as documentation homepage available at: [docs.company.com](docs.company.com)
- repo "beaver" - CircleCI orbs repo  available at: [docs.company.com/beaver](docs.company.com)

`Owl` deploys to the `root` directory if webserver (s3 bucket), `beaver` deploys to `beaver` directory.

This is the reason, why I am looking for relative HTML link.

Root directory:
<img width="1351" alt="Screen Shot 2021-09-26 at 07 47 33" src="https://user-images.githubusercontent.com/51825189/134795294-516fd06e-eb7a-4276-8be2-9f62471968c9.png">

Beaver directory:
<img width="1395" alt="Screen Shot 2021-09-26 at 07 49 02" src="https://user-images.githubusercontent.com/51825189/134795324-721f88d6-4c4b-4faf-af4c-efba9699e5dc.png">
